src/installer/corehost/build.sh

```diff
@@ -82,7 +87,7 @@ __DistroRidLower="$(echo $__DistroRid | tr '[:upper:]' '[:lower:]')"
 __BinDir="$__RootBinDir/bin/$__DistroRidLower.$__BuildType"
 __IntermediatesDir="$__RootBinDir/obj/$__DistroRidLower.$__BuildType"
 
-export __BinDir __IntermediatesDir __CoreClrArtifacts
+export __BinDir __IntermediatesDir __CoreClrArtifacts __NativeLibsArtifacts
 
 __CMakeArgs="-DCLI_CMAKE_HOST_VER=\"$__host_ver\" -DCLI_CMAKE_COMMON_HOST_VER=\"$__apphost_ver\" -DCLI_CMAKE_HOST_FXR_VER=\"$__fxr_ver\" $__CMakeArgs"
 __CMakeArgs="-DCLI_CMAKE_HOST_POLICY_VER=\"$__policy_ver\" -DCLI_CMAKE_PKG_RID=\"$__DistroRid\" -DCLI_CMAKE_COMMIT_HASH=\"$__commit_hash\" $__CMakeArgs"
```

Comment:
```suggestion
__CMakeArgs="-DNATIVE_LIBS_LOCATION=\"$__NativeLibsArtifacts\" -DCLI_CMAKE_HOST_POLICY_VER=\"$__policy_ver\" -DCLI_CMAKE_PKG_RID=\"$__DistroRid\" -DCLI_CMAKE_COMMIT_HASH=\"$__commit_hash\" $__CMakeArgs"
```

Reply:
Thanks! I will try switching coreclr artifacts path to this too. It uses the same pattern.

Reply:
Can the same be done for windows/.bat case? It looks quite different.

Reply:
It looks like windows got out of date and didn't unify with the rest of the native builds. You'll have to update the `gen-buildsys-win.bat` file and add the variables to `%__ExtraCmakeParams%` (with the same value as the Linux case) here: https://github.com/dotnet/runtime/blob/9715a9c2b43a9111e91c7997fc2cb838e5250969/src/installer/corehost/Windows/gen-buildsys-win.bat#L44

Reply:
@jkoritzinsky - the artifact paths are CMakeArgs now in both Windows and Unix.